### PR TITLE
CHAOS 3624 Sandbox certification

### PR DESCRIPTION
**Pull request by SmritiSatya**
Sandbox certification tutorial 
[Sandbox](https://659ba71c4cfc7921c57e8d8d--harness-developer.netlify.app/docs/chaos-engineering/ce-onboarding-guide/run-experiments-in-sandbox)
[Developer certification](https://659ba71c4cfc7921c57e8d8d--harness-developer.netlify.app/docs/chaos-engineering/ce-onboarding-guide/developer-certification)

### CODE REVIEW

**Comment by ravilach** [comment]
@SmritiSatya we don't directly link to university.harness.io, can you link to the HDH page; https://developer.harness.io/certifications/chaos-engineering?lvl=developer to register please?

File: src/components/LandingPage/data/chaosEngineeringData.tsx [hunk 1]
```diff
@@ -103,4 +103,18 @@ export const CEList: CardItem[] = [
     time: "15min",
     link: "/tutorials/chaos-experiments/integration-with-harness-cd",
   },
+  {
+    title: "Improve resilience of Podtato head application in sandbox",
```
Comment (sagarkrsd) [inline src/components/LandingPage/data/chaosEngineeringData.tsx:107]: The tutorial heading could be a little different which gives a feeling that users can do hands-on on Chaos experiments using Chaos Sandbox without any prerequisites like a k8s cluster, VM, etc.

File: tutorials/chaos-experiments/improve-resilience-of-podtato-head-app-in-sandbox.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+![enter details](./static/sandbox/create-account.png)
+
+3. You will receive a verification email that will lead you to the page with various modules.
+
+![verify](./static/sandbox/verify-email.png)
+
+
+## Execute pod delete on the Podtato head app
+
+Once you have created an account with Harness, click **Get started** of the **Chaos Engineering** module.
```
Comment (sagarkrsd) [inline tutorials/chaos-experiments/improve-resilience-of-podtato-head-app-in-sandbox.md:29]: Don't we need to start `Free trial` or something?

**Review by sagarkrsd — APPROVED** [review]
Thanks @SmritiSatya , LGTM!

**Review by SahilKr24 — APPROVED** [review]
Looks good to me

**Pull request merged**